string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,14 @@
+/* exported isUpperCased */
+/* Use toUppercase() method to capitalize each character on word and store the uppercase version
+   Compare the uppercase version to original word
+   If the uppercase version is the same as the original word, return true
+   If not, return false */
+function isUpperCased(word) {
+  var uppercasedWord;
```

Comment:
This is not necessary in JavaScript, you can assign a value as you declare a variable.